scipy/io/netcdf.py

```diff
@@ -989,4 +994,4 @@ def __setitem__(self, index, data):
 
 
 NetCDFFile = netcdf_file
-NetCDFVariable = netcdf_variable
+NetCDFVariable = netcdf_variable
```

Comment:
> By any means it appears that the issue this pull request meant to solve has already been solved by a past patch without resolving the issue.

@CallumJHays I'm not sure exactly what you mean here? Which other patch/PR, and what issue exactly?